web/setup.py

```diff
@@ -180,11 +180,14 @@ def add_external_user(username: str,
         ManageUsers.create_user(data, console, json)
 
     @app.command()
-    def delete_user(username: str, auth_source: AuthType = AuthType.internal):
+    def delete_user(username: str,
+                    auth_source: AuthType = AuthType.internal,
+                    auto_confirm: Annotated[Optional[bool],
```

Comment:
Please fix the PEP-8 issue. Check https://github.com/pgadmin-org/pgadmin4/actions/runs/7868821161/job/21470109856?pr=7190 for the same.